src/commands/quiz_event.ts

```diff
@@ -0,0 +1,188 @@
+import { CommandInteraction, DMChannel, GuildMember, HexColorString, SlashCommandBuilder } from 'discord.js'
+import { Command, He4rtClient } from '@/types'
+import { QUIZ_EVENT } from '@/defines/ids.json'
+import QUIZ from '-/commands/quiz_event.json'
+import { START_CODE_CHALLENGE } from '@/defines/commands.json'
+import { TIMEOUT_ANSWER, TIMEOUT_COMMAND_STRING, COLORS } from '@/defines/values.json'
+import { getChannel, isValidId, reply, sendInDM } from '@/utils'
+import { embedTemplate } from '@/utils'
+import { checkUserEventEntry, claimEventReward, getActiveEvent, getEventQuizzesById } from '@/http/firebase'
+const nextTextMessage = async (dm: DMChannel, interaction: CommandInteraction): Promise<string> => {
+  try {
+    const result = await dm.awaitMessages({
+      filter: (m) => m.author.id === interaction.user.id,
+      time: TIMEOUT_ANSWER,
+      max: 1,
+    })
+
+    return result.first()!.content
+  } catch (e) {
+    return TIMEOUT_COMMAND_STRING
+  }
+}
+
```

Comment:
Faz sentido usarmos um try catch se não estamos usando? Qual erro poderia dar nesse trecho? 

Se for algo não possivel de se tratar, ok. Mas caso contrario pode deixar, apenas renomeie a variavel `e` para `exception` por mais que não use é importante dar nomes declararitivos.